package.json

```diff
@@ -12,16 +12,14 @@
     "@hapi/joi": "^17.1.1",
     "@naturalcycles/js-lib": "^13.0.0",
     "@naturalcycles/time-lib": "^3.0.1",
-    "@types/hapi__joi": "^17.1.0",
```

Comment:
Actually, they were intentionally exported as `dependencies`. So, the consumers of the package don't need to install them